pkg/ciliumenvoyconfig/cec_manager.go

```diff
@@ -127,7 +127,7 @@ func (r *cecManager) addK8sServiceRedirects(resourceName service.L7LBResourceNam
 
 		// Tell service manager to redirect the service to the port
```

Comment:
nit: maybe enrich that comment that if configured, only a subset of frontend ports is forwarded to the L7 proxy.

Reply:
leaving this for future enhancement